src/server/svr_mail.c

```diff
@@ -145,8 +145,8 @@ svr_mailowner_id(char *jid, job *pjob, int mailpoint, int force, char *text)
 
 			/* see if user specified mail of this type */
 
-			if (pjob->ji_wattr[(int)JOB_ATR_mailpnts].at_flags & ATR_VFLAG_SET) {
-				if (strchr(pjob->ji_wattr[(int)JOB_ATR_mailpnts].at_val.at_str,
+			if (is_jattr_set(pjob, JOB_ATR_mailpnts)) {
```

Comment:
Here too.

Reply:
again, can't merge these, it'll change the logic